src/main/java/com/matchus/domains/user/domain/User.java

```diff
@@ -114,4 +114,8 @@ public void checkPassword(PasswordEncoder passwordEncoder, String credentials) {
 		}
 	}
 
+	public void deactivateUser() {
```

Comment:
```suggestion
	public void deactivate() {
```

User 엔티티 내부라 User 는 빼도 될 거 같습니다!